### ISSUE TITLE
sparkjob能否人工指定service-account?

### ISSUE THREAD

**Issue by Alienfeel** [kubeflow/arena#612]
```
apiVersion: "sparkoperator.k8s.io/v1beta2"
kind: SparkApplication
metadata:
  name: {{ .Release.Name }}
  labels:
    version: 2.4.5
    release: {{ .Release.Name }}
    app: {{ template "sparkjob.name" . }}
    createdBy: "SparkJob"
spec:
  type: {{ .Values.Lang }}
  mode: cluster
  image: {{ .Values.Image }}
  imagePullPolicy: Always
  mainClass: {{ .Values.MainClass }}
  mainApplicationFile: {{ .Values.Jar }}
  sparkVersion: "2.4.5"
  restartPolicy:
    type: Never
  driver:
    cores: {{ .Values.Driver.CPURequest }}
    memory: {{ .Values.Driver.MemoryRequest }}
    labels:
      **version: 2.4.5**
      release: {{ .Release.Name }}
      app: {{ template "sparkjob.name" . }}
      createdBy: "SparkJob"
    **serviceAccount: spark**
  executor:
    cores: {{ .Values.Executor.CPURequest }}
    instances: {{ .Values.Executor.Replicas }}
    memory: {{ .Values.Executor.MemoryRequest }}
    labels:
      version: 2.4.5
      release: {{ .Release.Name }}
      app: {{ template "sparkjob.name" . }}
      createdBy: "SparkJob"
```

1. serviceAccount能否人工指定？
2. sparkVersion 能否放到values.yaml中配置？

**Comment by ChenYi015** [comment]
Now service account can be specified by `--driver-service-account` and spark version can be specified by `--spark-version`:

```bash
$ bin/arena submit spark --help
Submit a common spark application job.

Usage:
  arena submit sparkjob [flags]

Aliases:
  sparkjob, spark

Flags:
  -a, --annotation stringArray           the annotations, usage: "--annotation=key=value" or "--annotation key=value"
      --driver-cpu-request int           cpu request for driver pod (default 1)
      --driver-memory-request string     memory request for driver pod (min is 500m) (default "500m")
      --driver-service-account string    the service account of driver pod (default "spark")
      --executor-cpu-request int         cpu request for executor pod (default 1)
      --executor-memory-request string   memory request for executor pod (min is 500m) (default "500m")
  -h, --help                             help for sparkjob
      --image string                     the docker image name of training job (default "spark:3.5.0")
      --jar string                       jar path in image (default "local:///opt/spark/examples/jars/spark-examples_2.12-3.5.0.jar")
  -l, --label stringArray                specify the label
      --main-class string                main class of your jar (default "org.apache.spark.examples.SparkPi")
      --name string                      override name
      --replicas int                     the executor's number to run the distributed training. (default 1)
      --spark-version string             the spark version of spark job (default "3.5.0")
```

**Comment by ChenYi015** [comment]
Closed by #1112 .
